public/api/conf/2.0/application.raml

```diff
@@ -40,12 +40,12 @@ documentation:
           required: true
           example: application/xml; charset=UTF-8
         X-Badge-Identifier:
-          description: Required only for submission of declarations via CSPs. Represents the end-user who is interacting with the subscribed application. Must be numeric or upper case letters with a minimum length of 6 and a maximum length of 12.
```

Comment:
Put or both in brackets - same for all places where badge and submitter header are used in RAML

> (or both)

Reply:
brackets added